### ISSUE TITLE
plant_omni does not have dim=3

### ISSUE THREAD

**Issue by donghyunkm** [kevinjohncutler/omnipose#121]
Hello,

As I was going through the notebooks, I found an error with the 'plant_omni" pretrained model. 

The code and this [link](https://omnipose.readthedocs.io/models.html?highlight=plant_omni) indicates that 'plant_omni' should have dim=3 but using dim=3 results in "failed to load model Error(s) in loading state_dict for CPnet" and additional errors during segmentation. 

When I create a 'plant_omni' model without the dim argument, the model is shown to have model.dim = 2.

Thus, I believe the checkpoint used to internally load the pretrained model may be incorrect/outdated?

Attached are relevant screenshots.

Thank you for your help!

<img width="2021" alt="Image" src="https://github.com/user-attachments/assets/b28008ee-1ed2-4970-82ae-bd23b6fda125" />
<img width="1284" alt="Image" src="https://github.com/user-attachments/assets/2e989aa4-34e1-445e-a706-171aeb382f6c" />
<img width="2019" alt="Image" src="https://github.com/user-attachments/assets/55d4b4f6-4781-4c3e-b7a0-7bf4651a0fe1" />
